src/client/legacy.rs

```diff
@@ -1347,6 +1348,22 @@ impl Builder {
         self
     }
 
+    /// Provide a timer to execute background HTTP2 tasks
+    ///
+    /// See the documentation of [`h2::client::Builder::timer`] for more
+    /// details.
+    ///
+    /// [`h2::client::Builder::timer`]: https://docs.rs/h2/client/struct.Builder.html#method.timer
+    #[cfg(feature = "http2")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "http2")))]
+    pub fn http2_timer<M>(&mut self, timer: M) -> &mut Self
```

Comment:
Doesn't the pool need a timer too? Should we just call this method `timer()`?

Reply:
> Ah, is that because you're setting some of the HTTP/2 keep-alive options?

Ah... that is it. I didn't make that connection.

I updated the name to `timer()` and opened https://github.com/hyperium/hyper/issues/3167 to track making Pool use it as well. I am hoping to pick up that task as well but won't be able to get to it for a few days at least so would prefer it separate

Reply:
> I updated the name to timer()

Did you mean to push a new commit? 😅

Reply:
Oops, should be pushed now. Thanks!